docs/guidance/prometheus-grafana.md

```diff
@@ -0,0 +1,191 @@
+# Ray Cluster: Monitoring with Prometheus & Grafana
+
+This section will describe how to monitor Ray Clusters in Kubernetes using Prometheus & Grafana.
+
+If you do not have any experience with Prometheus and Grafana on Kubernetes, I strongly recommend you watch this [YouTube playlist](https://youtube.com/playlist?list=PLy7NrYWoggjxCF3av5JKwyG7FFF9eLeL4).
+
+## Step 1: Create a Kubernetes cluster with Kind.
+
+```sh
+kind create cluster
+```
+
+## Step 2: Install Kubernetes Prometheus Stack via Helm chart
+
+```sh
+# Path: kuberay/
+./install/prometheus/install.sh
+
+# Check the installation
+kubectl get all -n prometheus-system
+
+# (part of the output)
+# NAME                                                  READY   UP-TO-DATE   AVAILABLE   AGE
+# deployment.apps/prometheus-grafana                    1/1     1            1           46s
+# deployment.apps/prometheus-kube-prometheus-operator   1/1     1            1           46s
+# deployment.apps/prometheus-kube-state-metrics         1/1     1            1           46s
+```
+* KubeRay provides an [install.sh script](../../install/prometheus/install.sh) to install the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) chart and related custom resources, including **ServiceMonitor** and **PodMonitor**, in the namespace `prometheus-system` automatically.
+
+## Step 3: Install a KubeRay operator
+
+* Follow this [document](../../helm-chart/kuberay-operator/README.md) to install the latest stable KubeRay operator via Helm repository.
+
+## Step 4: Install a RayCluster
+
+```sh
+helm install raycluster kuberay/ray-cluster --version 0.4.0
+
```

Comment:
Consider explicitly adding a command here for how to get the RAYCLUSTER_HEAD_POD needed in the next step.   I used `kubectl get pods`

Reply:
Maybe also indicate that the user needs to watch this command until the pod is RUNNING (though the error message is pretty easy to understand: `error: unable to forward port because pod is not running. Current status=Pending`)

Reply:
Updated. https://github.com/ray-project/kuberay/pull/895/commits/b990af9fac79ff070e0024429efef4bd0f0141e4